layers/+lang/typescript/packages.el

```diff
@@ -72,6 +72,8 @@
                 (tide-setup)
                 (flycheck-mode +1)
                 (eldoc-mode +1)
+                (define-key spacemacs-web-mode-map (kbd "g g") 'tide-jump-to-definition)
+                (define-key spacemacs-web-mode-map (kbd "g u") 'tide-references)
```

Comment:
Also, please use the new jumping API, don't bind `SPC g g` directly. See other layers for examples.